docs/en/interfaces/http_interface.md

```diff
@@ -130,14 +130,13 @@ POST 'http://localhost:8123/?query=DROP TABLE t'
 
 For successful requests that don't return a data table, an empty response body is returned.
 
-You can use compression when transmitting data.
+You can use the internal ClickHouse compression format when transmitting data. The compressed data has a non-standard format, and you will need to use the special clickhouse-compressor program to work with it (it is installed with the clickhouse-client package).
```

Comment:
Bad merge.

Reply:
There is no bad merge. English version is exactly the same as russian one in this place.